<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskinfodlgClass</class>
 <widget class="QDialog" name="taskinfodlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add New Download</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="title">
       <string>Gerneral</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>URL:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Split file</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="tid_g_le_pb_show_dir">
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="tid_g_cb_seache_mirror">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="tid_g_le_referrer"/>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSpinBox" name="tid_g_sb_split_file">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>part</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Find alternate URLs through the mirror list file or Ftp Search</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="tid_g_le_le_rename"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Referrer:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="tid_g_cb_save_to">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>C:/NGDownload</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Rename:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="tid_g_le_cb_category">
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="modelColumn">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="tid_g_le_pb_category_info">
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Save to:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="tid_g_le_url">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>http://localhost/mtv.wmv</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="0" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Login to server:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Username:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="tid_g_le_le_user_name">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="tid_g_le_cb_login_to_server">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="tid_g_le_le_password">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maxLength">
                <number>32767</number>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Start</string>
             </property>
             <widget class="QRadioButton" name="tid_g_le_rb_schedule">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>90</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Schedule</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="tid_g_le_rb_manual">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>90</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Manual</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="tid_g_le_rb_immediately">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>90</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Immeditely</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tid_g_le_pb_save_as_default">
             <property name="text">
              <string>Save as default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Comment:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="tid_g_le_te_comment"/>
           </item>
           <item>
            <widget class="QPushButton" name="tid_g_le_pb_show_site_property">
             <property name="text">
              <string>Site Property</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Alternate URLs</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="tid_au_lb_url">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>The Task URL , dynamic update on run.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="tid_au_lw_alter_urls">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="tid_au_pb_add">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tid_au_pb_delete">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tid_au_pb_ok">
           <property name="text">
            <string>&amp;Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ftp</string>
         </property>
         <widget class="QCheckBox" name="tid_ad_cb_down_subdir_from_ftp">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>381</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Download &amp;subdirectories from Ftp</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="tid_ad_cb_create_subdir_locally">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>381</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Create sub&amp;directories locally</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="tid_ad_cb_create_category">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>281</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Create &amp;category</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Proxy</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Http:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="tid_ad_cb_proxy_type_http">
              <item>
               <property name="text">
                <string>Direct Connection</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Ftp:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="tid_ad_cb_proxy_type_ftp">
              <item>
               <property name="text">
                <string>Direct Connection</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Rtsp/MMS:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="tid_ad_cb_proxy_type_media">
              <item>
               <property name="text">
                <string>Direct Connection</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>no text needed </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Reject</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tid_g_le_url</tabstop>
  <tabstop>tid_g_cb_seache_mirror</tabstop>
  <tabstop>tid_g_le_referrer</tabstop>
  <tabstop>tid_g_le_cb_category</tabstop>
  <tabstop>tid_g_le_pb_category_info</tabstop>
  <tabstop>tid_g_cb_save_to</tabstop>
  <tabstop>tid_g_le_pb_show_dir</tabstop>
  <tabstop>tid_g_le_le_rename</tabstop>
  <tabstop>tid_g_sb_split_file</tabstop>
  <tabstop>tid_g_le_cb_login_to_server</tabstop>
  <tabstop>tid_g_le_le_user_name</tabstop>
  <tabstop>tid_g_le_le_password</tabstop>
  <tabstop>tid_g_le_rb_manual</tabstop>
  <tabstop>tid_g_le_rb_immediately</tabstop>
  <tabstop>tid_g_le_rb_schedule</tabstop>
  <tabstop>tid_g_le_te_comment</tabstop>
  <tabstop>tid_g_le_pb_show_site_property</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tid_au_lw_alter_urls</tabstop>
  <tabstop>tid_au_pb_add</tabstop>
  <tabstop>tid_au_pb_delete</tabstop>
  <tabstop>tid_au_pb_ok</tabstop>
  <tabstop>tid_ad_cb_down_subdir_from_ftp</tabstop>
  <tabstop>tid_ad_cb_create_subdir_locally</tabstop>
  <tabstop>tid_ad_cb_create_category</tabstop>
  <tabstop>tid_ad_cb_proxy_type_http</tabstop>
  <tabstop>tid_ad_cb_proxy_type_ftp</tabstop>
  <tabstop>tid_ad_cb_proxy_type_media</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tid_g_le_pb_save_as_default</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>taskinfodlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>taskinfodlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tid_g_le_cb_login_to_server</sender>
   <signal>clicked(bool)</signal>
   <receiver>tid_g_le_le_user_name</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tid_g_le_cb_login_to_server</sender>
   <signal>clicked(bool)</signal>
   <receiver>tid_g_le_le_password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
